jetty-http/src/main/java/org/eclipse/jetty/http/HttpGenerator.java

```diff
@@ -67,10 +67,11 @@
     private Boolean _persistent = null;
 
     private final int _send;
+    private final boolean _chunkEOFContent;
```

Comment:
I'm not keen on the boolean flag approach.  Firstly it has to be passed around in constructors etc. making it bit too much of a special case. Secondly it makes the entire server do this, when it  would be better to allow it to be done response by response.

My preference would be to use a `Transfer-Encoding: chunked` header set in the response fields by the application to act as a hint to the container that it should chunk if it can.  This is the same as we use `Connection: close` set by the application as a hint to close the connection.

In fact, this may have already been done in 9.4...